packages/mds-web-sockets/clients.ts

```diff
@@ -0,0 +1,23 @@
+import WebSocket from 'ws'
+
+export class Clients {
+  clientList: { [key: string]: WebSocket[] }
+
+  public constructor() {
+    this.clientList = { 'EVENTS': [], 'TELEMETRIES': [] }
+    this.saveClient = this.saveClient.bind(this)
+  }
+
+  public saveClient(entities: string[], client: WebSocket) {
+    console.log(entities.length)
+    entities.map(entity => {
+      console.log(entity)
+      console.log(this.clientList)
+      try {
+        this.clientList[entity].push(client)
```

Comment:
Isn't this just a normal JS array you're `push()`ing to? AFAIK that shouldn't throw anything.

Reply:
`clientList[entity]` may be undefined